src/main/java/org/jivesoftware/smackx/pubsub/PubSubIntegrationTest.java

```diff
@@ -915,4 +915,36 @@ public void transientNotificationOnlyNodeWithItemTest() throws NoResponseExcepti
             pubSubManagerOne.deleteNode(nodename);
         }
     }
+
+    /**
+     * Asserts that the server returns an 'item-not-found' error response when
+     * deleting a node that does not exist.
+     *
+     * <p>
+     * From XEP-0060 § 8.4.3.2:
+     * </p>
+     * <blockquote> If the requesting entity attempts to delete a node that does not
+     * exist, the service MUST return an &lt;item-not-found/&gt; error.
+     * </blockquote>
+     * 
+     * @throws NoResponseException   if there was no response from the remote
+     *                               entity.
+     * @throws XMPPErrorException    if there was an XMPP error returned.
+     * @throws NotConnectedException if the XMPP connection is not connected.
+     * @throws InterruptedException  if the calling thread was interrupted.
+     */
+
+    @SmackIntegrationTest
+    public void deleteNonExistentNodeTest() throws NoResponseException, XMPPErrorException, NotConnectedException, InterruptedException {
+        final String nodename = "sinttest-publish-item-nodename-" + testRunId;
```

Comment:
It is confusing to give the node a name that refers to a different test. Change the name of the node to `"sinttest-delete-node-that-does-not-exist-" + testRunId` (or something similar).

Reply:
It is now changed